tsconfig.json

```diff
@@ -13,6 +13,6 @@
     "strict": true,
     "noImplicitAny": true
   },
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "include": ["src/**/*.ts", "test/**/*.ts", "models/**/*.ts"],
```

Comment:
We don't need to include `models` since they are just used to test. 😄

Reply:
...ok, but then why is it necessary to include `test/**/*.ts` since they are interpreted by `ts-jest`, right?

I pushed a version which do not include `test/**/*.ts`, `yarn test` seems to work even rebuilding project from scratch.

Reply:
That's true :palm_face:
You can add `models` 😄 
```
"include": ["src/**/*.ts", "test/**/*.ts", "models/**/*.ts"],
```

Reply:
Okay 😄 

https://github.com/bayesjs/bayesjs/pull/40